crates/ironrdp-web/src/error.rs

```diff
@@ -3,7 +3,7 @@ use wasm_bindgen::prelude::*;
 
 #[wasm_bindgen]
 #[derive(Clone, Copy)]
-pub enum IronRdpErrorKind {
+pub enum RemoteDesktopErrorKind {
```

Comment:
thought: Should we rename to `IronErrorKind`/`IronError`? Since we already have `iron_init`. Just discussing the idea, no need to rename in this PR.

Reply:
Fixed in https://github.com/Devolutions/IronRDP/pull/747